CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing
+
+We welcome pull requests from anyone.
+
+Fork, then clone the repo:
+
+    git clone git@github.com:your-username/aaas.git
```

Comment:
I'm not sure we need to tell them how to clone repos. Especially if they arent using SSH